boot.php

```diff
@@ -55,34 +55,14 @@
 	define('SIGTERM', 15);
 }
 
-/**
- * Depending on the PHP version this constant does exist - or not.
- * See here: http://php.net/manual/en/curl.constants.php#117928
- */
-if (!defined('CURLE_OPERATION_TIMEDOUT')) {
-	define('CURLE_OPERATION_TIMEDOUT', CURLE_OPERATION_TIMEOUTED);
-}
-
-if (!function_exists('exif_imagetype')) {
-	function exif_imagetype($file)
-	{
-		$size = getimagesize($file);
-		return $size[2];
-	}
-}
-
 /**
  * Returns the user id of locally logged in user or false.
  *
  * @return int|bool user id or false
```

Comment:
Please mark this method as deprecated
```suggestion
 * @return int|bool user id or false
 * @deprecated since version 2022.12, use Core\Session::getLocalUser() instead
```